releasenotes/2.1.0.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: releasenotes
+title: Release Notes for Version 2.1.0
+date: 2018-02-21 12:00:00
+release_notes: true
+permalink: /releasenotes/2.1.0/
+---
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Compute Improvements](#computeimprovements)
+1. [BlobStore Improvements](#blobstoreimprovements)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.1.0 represents an important milestone for the project. We are proud to support all major cloud providers in the marketplace with a mature, stable, codebase that is ready for production. You can [read the official announcement here](https://s.apache.org/jclouds210). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
```

Comment:
The link to the announcement thread is not correct.

Reply:
thanks, fixed

Reply:
> The link to the announcement thread is not correct.

Thanks for catching that, Ignasi!

Reply:
Is it ok now?